### DISCUSSION TITLE
Getting Started with GraphQL with Lucia Cerchie (Lunch & Learn November 12, 2021)

### DISCUSSION THREAD

**Discussion by bogdaaamn** [Virtual-Coffee/virtualcoffee.io#477] [Events]
Want to get started with GraphQL or better understand what it is and why its used? Lucia welcomes all experience levels and outlines the basics of GraphQL. Things like, "What is GraphQL?", "How do I write a query?", and "Where do I begin learning?" are all valid questions for this Lunch & Learn.

Thanks a lot Lucia @Cerchie for this talk! 

Recording available here: https://youtu.be/mUt0vojFyXY

**Resources from Lucia:** 
- https://graphql.org/ - official resource site
- https://moonhighway.com/ - Eve Porcello does free monthly trainings
- https://spec.graphql.org/October2021/ - “the spec”
- https://github.com/stepzen-samples/stepzen-graphql-api-mysql-db - open-source repo to practice with
- https://stepzen.com/docs/using-graphql - StepZen docs on using GQL
- https://stepzen.com/learn/build-with-graphql-studio - workshop on building an API with the StepZen GraphQL studio

**Questions from Lucia:** (feel free to answer below)
- What are you doing with GraphQL?
- What would you like to do with GraphQL?

**Comment by bogdaaamn** [comment]
I am really into the idea of not using `.map` in React anymore 👀

**Reply by Cerchie** [reply]
Yeah, thinking about it more, it would depend on the use-case because you might still want to map over returned arrays I think -- but you'd definitely see .filter less.

**Comment by Cerchie** [comment]
Here's another post that might be useful:
https://dev.to/stepzen/what-does-it-mean-for-graphql-to-be-a-runtime-anyway-1bpb
